source/Ship.cpp

```diff
@@ -4177,7 +4177,11 @@ void Ship::DoGeneration()
 		if(currentSystem)
 		{
 			double scale = .2 + 1.8 / (.001 * position.Length() + 1);
-			fuel += currentSystem->RamscoopFuel(attributes.Get("ramscoop"), scale);
+			double totalRamscoop = attributes.Get("ramscoop");
+			totalRamscoop += IsHyperspacing() ? 0. : (attributes.Get("velocity ramscoop") * velocity.Length() / 1.66);
```

Comment:
> I've scaled `velocity ramscoop` such that a ship velocity of 100 results in the same fuel generation as an equal value of the `ramscoop` attribute.

OK, so we want `fuel("velocity ramscoop" * velocity / c) = fuel("ramscoop")`, right?
`"velocity ramscoop" = "ramscoop"` and `velocity = 100`, so we have `100 / c = 1`, `c = 100`.
```suggestion
			totalRamscoop += IsHyperspacing() ? 0. : (attributes.Get("velocity ramscoop") * velocity.Length() / 100.);
```